migrations/20200531185104-solutions-metaform.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+  async up(db) {
+    
+    global.migrationMsg = "Add solution metaform information";
+
+    let metaForm = [
+      {
+        field : "name",
+        label : "Name",
+        value : "",
+        visible : true,
+        editable : true,
+        input : "text",
+        validation : {
+          required : true
+        },
+        min : "",
+        max : ""
+      },{
+        field : "description",
+        label : "Description",
+        value : "",
+        visible : true,
+        editable : true,
+        input : "text",
+        validation : {
+          required : true
+        },
+        min : "",
+        max : ""
+      },{
+        field : "program",
+        label : "Program",
+        value : "",
+        visible : true,
+        editable : true,
+        input : "selectOrText",
+        validation : {
+          required : true
+        },
+        min : "",
+        max : ""
+      },
+      {
+        field : "entities",
+        label : "Select Entity",
+        value : "",
+        visible : true,
+        editable : true,
+        input : "text",
+        validation : {
+          required : true
+        },
+        min : "",
+        max : ""
+      }
+    ];
+
+    let solutionsType = [
+      "observation",
+      "institutional",
+      "individual"
+    ];
+
+    let result = [];
+
+    for ( let type = 0 ; type < solutionsType.length; type++ ) {
+
+      let form = {};
+
+      if( solutionsType[type] === "observation") {
+        form["name"] = "observationSolutionsMetaForm";
```

Comment:
@aman-tunerlabs - Please set the name as observationMetaFormV2 and institutionalAssessmentMetaForm